.github/workflows/publish-command.yml

```diff
@@ -288,18 +288,25 @@ jobs:
           echo "IMAGE_VERSION=$(_get_docker_image_version ${DOCKERFILE})" >> $GITHUB_ENV
       - name: Prepare Sentry
         if: startsWith(matrix.connector, 'connectors')
-        run: |
-          curl -sL https://sentry.io/get-cli/ | bash || echo "sentry cli already installed"
+        uses: nick-fields/retry@v2
+        with:
+          timeout_seconds: 15
```

Comment:
would it be fine to retry without a timeout for this?

Reply:
Done, I have removed timeout for this.